### PR TITLE
Prepare test image for OpenShift test lane.

### PR DESCRIPTION
**Pull request by tomob**
Signed-off-by: Tomasz Baranski <tbaransk@redhat.com>

**What this PR does / why we need it**:

This PR prepares a docker image with functional tests binary for use in OpenShift CI

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

This is build to mimic Kubevirt's OS CI image. I'm not 100% sure what I'm doing.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/ci/Dockerfile.ci [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+ENV KUBEVIRT_RUN_UNNESTED=true
+
+RUN source /etc/profile.d/gimme.sh && \
+    export GOPATH="/go" && \
+    go mod vendor && \
+    ./hack/ci/build.sh
+
+ENTRYPOINT [ "/entrypoint-bazel.sh" ]
+CMD ["./hack/ci/test.sh"]
```
Comment (awels) [inline hack/ci/Dockerfile.ci:14]: Doesn't look like hack/ci/test.sh exists?

Comment (tomob) [reply hack/ci/Dockerfile.ci:14]: Fixed

**Review by awels — APPROVED** [review]
/lgtm
/approve

**Pull request merged**